tree/master/cloud/src/solution/Microsoft.Legal.MatterCenter.jobs/Functions.cs

```diff
@@ -132,8 +132,16 @@ public class Functions
                                 foreach(string userName in userNames)
                                 {
                                     if(!string.IsNullOrWhiteSpace(userName))
-                                    {
-                                        email.ToRecipients.Add(userName);
+                                    {

```

Comment:
If the new matter contains, external user, we should not send the notification email